### ISSUE TITLE
Old issue are not included in codeclimate.json and this mark them as resolved in Gitlab

### ISSUE THREAD

**Issue by dzamlo** [gabrie-allaigre/sonar-gitlab-plugin#182]
The generated `codeclimate.json` only include new issues. The issue with that, is that the old issue that are not resolved are marked as resolved in the Merge request  even if they are not fixed. The `codeclimate.json` file should include all issues.

**Comment by Stan0304** [comment]
I agree too. CodeClimate does not implement such thing as analysis on new code and GitLab expects a `codeclimate.json` file that includes all the issues all the time. 
But it seems already to be the case with the last versions available and this issue should be closed.
